packages/rest-explorer/src/rest-explorer.keys.ts

```diff
@@ -4,13 +4,17 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {BindingKey} from '@loopback/context';
+import {RestExplorerComponent} from './rest-explorer.component';
 import {RestExplorerConfig} from './rest-explorer.types';
 
 /**
  * Binding keys used by this component.
  */
 export namespace RestExplorerBindings {
-  export const CONFIG = BindingKey.create<RestExplorerConfig>(
-    'rest-explorer.config',
```

Comment:
As discussed before, this is changing the actual string value of the binding key used for REST Explorer config.

Consider the following code:

```ts
this.bind('rest-explorer.config').to({
  path: '/openapi/ui',
});      
```

At the moment, this code works well and configures REST Explorer as intended.

With your changes in place, the code no longer works and the user is not notified about the problem.

Should we treat your pull request as a breaking change then? Perhaps we can claim that the actual key name `rest-explorer.config` was undocumented and therefore changes of that string are not considered as breaking backwards compatibility?

@strongloop/loopback-maintainers @strongloop/loopback-next thoughts?

Reply:
I'm leaning toward the claim that `rest-explorer.config` is an internal string.

Reply:
The other option is to enhance `@config` to accept the target binding key to override the one from the enclosing binding.

Reply:
> I'm leaning toward the claim that rest-explorer.config is an internal string.

Yeah I would say +1, did a quick search in our existing doc, we don't mention `rest-explorer.config` anywhere, so unless users jump into the source code, they should always use the imported const `RestExplorerBindings.CONFIG`.

Reply:
Fair enough, let's treat `rest-explorer.config` as an internal string :+1: